### ISSUE TITLE
Style/AsciiComments default Enabled is true

### ISSUE THREAD

**Issue by esotericpig** [rubocop/rubocop#9674]
I know that you can change it to be disabled, but it seems a bit odd that `Style/AsciiComments` is enabled by default. I only noticed when working on a Japanese project, and I put Japanese in a comment.

I understand wanting this for identifiers, constants, etc., but why for comments?

If it's to enforce English comments, I don't think it should be on by default. It sends the message that RuboCop is, by default, for English speakers and not for the global audience.

If it's to enforce Ascii encoding for the file, that is rare nowadays (since Ruby v2), so it seems like AsciiComments should still be defaulted to disabled (since the majority of Rubyists now have source files defaulted to UTF-8).

Thanks for the consideration, and I understand if it will remain defaulted to enabled.

**Comment by marcandre** [comment]
I agree. This also precludes emojis in comments.

**Comment by dvandersluis** [comment]
#72 for historical context.

**Comment by bbatsov** [comment]
Yeah, the idea was mostly to make sure that people stick to English in comments, with the assumption they'd likely have an international team working on the codebase. The cop was inspired partly from my own experience working in German/Austrian company where many people didn't know English yet the German and Austrian devs would often use German for identifiers and comments making it hard for people like me to understand what they meant. :-) OSS libraries are commonly documented in English as well.

I'm fine with retiring this cop, as agree it's somewhat controversial, plus detecting what's really English is pretty complex. Still, this can't happen before RuboCop 2.0. I guess we should also change the cop's name and improve it's documentation.

**Comment by esotericpig** [comment]
I guess in a somewhat related note the enabled-by-default `Style/NumericLiterals` requires numbers to be grouped by 3s by default:

```
100_000
```

But Japanese (and probably Chinese as well?), like to group it in 4s:

```
10_0000
```

If want to make it more international-friendly, maybe either turn this off by default or add a `groupby` option to the cop.

Actually, I ran into this when testing with Chinese phone numbers, which are grouped by 4 in mainland China:

```
phone = 12_3456_7890
```
